projects/netgrif-components-core/src/lib/data-fields/file-list-field/file-list-default-field/abstract-file-list-default-field.component.ts

```diff
@@ -167,6 +168,17 @@ export abstract class AbstractFileListDefaultFieldComponent extends AbstractBase
             }
         }
 
+        if (this.dataField.allowTypes && this.dataField.allowTypes?.length > 0) {
```

Comment:
This check is repeated from file-default-field. It can be extracted to a file utility class.

Reply:
done